### PR TITLE
Add support questions in the contributing docs

### PR DESCRIPTION
**Pull request by silvenon**
Is this ok?

### CODE REVIEW

**Comment by arthurvr** [comment]
I like this. :+1: 

Although, shouldn't this be on yeoman.io instead of it being in one generators contributing.md?

**Comment by arthurvr** [comment]
The Yeoman IRC channel on freenode is another nice place for support.

**Comment by sindresorhus** [comment]
> Although, shouldn't this be on yeoman.io instead of it being in one generators contributing.md?

Both.

> The Yeoman IRC channel on freenode is another nice place for support.

:+1:

**Comment by silvenon** [comment]
Added IRC info. Merge?

File: contributing.md [hunk 1]
```diff
@@ -3,3 +3,12 @@ See the [contributing docs](https://github.com/yeoman/yeoman/blob/master/contrib
...
 
 If not, `generator-gulp-webapp` is fork-friendly and you can always maintain a custom version which you `npm install && npm link` to continue using via `yo gulp-webapp` or a name of your choosing.
+
+## Support Questions
+
+We frequently get issues where people *modified* something after they finished scaffolding and things weren't working properly. These are classified as **support** questions; they are related to this generator, but not something we did wrong. It's best to ask those on [Stack Overflow] with tags `#yeoman` and `#gulp` instead of opening an issue here. You can also ask questions in our IRC channel (`#yeoman` on [Freenode]).
+
+If the issue has already been opened before it turned out to be a support question, feel free to paste the link to the Stack Overflow question, so we can answer it.
+
+[stack overflow]: (http://stackoverflow.com)
```
Comment (arthurvr) [inline contributing.md]: > `(http://stackoverflow.com)`

remove the parentheses

**Comment by arthurvr** [comment]
sup with the CI?

**Comment by silvenon** [comment]
Not sure, I added `[ci skip]` now, so it won't even start. I guess it's choking because I created a branch here instead of my own fork.

**Comment by arthurvr** [comment]
Thanks.

**Pull request merged**